src/partials/languages/en.html

```diff
@@ -53,5 +53,6 @@
   "source.file.date.updated": "Last update",
   "source.file.date.created": "Created",
   "tabs.title": "Tabs",
-  "toc.title": "Table of contents"
+  "toc.title": "Table of contents",
+  "top.title": "Back to Top"
```

Comment:
Sorry, one more thing - I've overseen. It should be "Back to top", not "Back to Top".